cloud/pkg/cloudhub/servers/httpserver/server.go

```diff
@@ -45,6 +45,7 @@ func StartHTTPServer() {
 	router := mux.NewRouter()
 	router.HandleFunc("/edge.crt", edgeCoreClientCert).Methods("GET")
 	router.HandleFunc("/ca.crt", getCA).Methods("GET")
+	router.HandleFunc("/isCloudcoreReady", electionHandler).Methods("GET")
```

Comment:
Could we simpify this endpoint as `/ready`?

Reply:
OK.Changed.